### PR TITLE
Update Unit Test for IP Utils Diff Config 

### PR DESCRIPTION
**Pull request by kar-rahul-aws**
Update Unit Test for IP Utils Diff Config 

Description
-----------
Changed test for prvProcessNetworkDownEvent to accomodate End Point changes 

Test Steps
-----------

Related Issue
-----------

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kar-rahul-aws** [comment]
bot run uncrustify

**Review by moninom1 — APPROVED** [review]

File: test/unit-test/FreeRTOS_IP_Utils_DiffConfig/FreeRTOS_IP_Utils_DiffConfig_utest.c [hunk 1]
```diff
@@ -83,24 +88,35 @@ void test_pxPacketBuffer_to_NetworkBuffer( void )
...
+void test_prvProcessNetworkDownEvent_Pass_DHCP_Enabled( void )
 {
     NetworkInterface_t xInterface;
     NetworkEndPoint_t xEndPoint;
 
-    xCallEventHook = pdFALSE;
+    xCallEventHook = pdFALSE;  
 
-    xInterfaces[ 0 ].pfInitialise = xNetworkInterfaceInitialise_CMockExpectAndReturn;
+    xInterface.pfInitialise = xNetworkInterfaceInitialise_test;
```
Comment (moninom1) [inline test/unit-test/FreeRTOS_IP_Utils_DiffConfig/FreeRTOS_IP_Utils_DiffConfig_utest.c:98]: Why not directly assign pdTRUE? Why is function needed here?

Comment (moninom1) [inline test/unit-test/FreeRTOS_IP_Utils_DiffConfig/FreeRTOS_IP_Utils_DiffConfig_utest.c:98]: Why is this needed?
